Update on the chapter on types of sensitive data (#2870)
alan-turing-institute/the-turing-way: book/website/project-design/sdp/types.md

* **Medium sensitivity / restricted data** - It could include details like personal data that is not classified as 'sensitive', student/alumni contact details, staff contact details. This kind of data is usually for internal usages only, and if compromised or destroyed in an unauthorized way, would not be as impactful as the former on the organizations/individuals involved.
* **Low sensitivity / internal data** - It could include details like non-confidential internal correspondence, working group minutes, internal policies and procedures. This kind of data could also comprise of data intended for public use, like public website contents or press releases.

According to the information guidelines provided by [Imperva](https://www.imperva.com/learn/data-security/data-classification/), data could be classified based on content, context or user-based. 
* **Content-based classification** - This type of classification would involve inspecting and interpreting files and documents looking for sensitive information while reviewing them, and classifying based on that.

> * **Context-based classification** - This type of classification would involve looking into the metadata to classify, such as the application which created the file, person/people involved in its creation, or the location of its authorization. Some questions one can ask while looking for this classification are:

Reviewer: ```suggestion
* **Context-based classification** -  involves looking into the metadata to classify, such as the application which created the file, person/people involved in its creation, or the location of its authorization. 
Some questions one can ask while looking for this classification are:
```

  * How is the data being used? 
  * Who has the access to it? 
  * When are they accessing it? 
* **User-based classification** - This type of classification would involve the expertise of judgement made by one or more knowledgeable users in classifying the file or document. Hence, they would be advising on specifying how sensitive the data is and make necessary changes or edits before they release it.

> In terms of research, as per the detailed guidelines provided by various universities across the UK (and elsewhere) abiding by the Data Protection Act, the most common reasons data falls under the high sensitivity data are:

Reviewer: ```suggestion
In terms of research the most common reasons data falls under the high sensitivity data are:
```
You already mention the laws/frameworks above, so I don't think its needed to repeat that info here.

* Research data containing personal identifying information, 'personal data' and special categories data as defined in UK and European data protection legislation. Such as involvement of human participants, particularly where the research involves sensitive personal data such as health records
* Involvement of commercial collaborators, particularly where the data could be construed as competitive intelligence
* Data relating to species of plants or animals where the release of data may adversely affect rare or endangered species
* Data likely to harm an individual or community or have a significant negative public impact if released
* Working under the terms of a non-disclosure agreement

> Organizations usually share guidances and links on storing sensitive data, controlling access to sensitive data, encrypting sensitive data, anonymising sensitive data, transferring sensitive data, and disposing of sensitive data. Adhering to the safety protocols are highly essential for ensuring data safety and security.

Reviewer: ```suggestion
Organizations usually share guidances and links on storing sensitive data, controlling access to sensitive data, encrypting sensitive data, anonymising sensitive data, transferring sensitive data, and disposing of sensitive data. 
Adhering to the safety protocols are highly essential for ensuring data safety and security.
```
Here it might be good to refer to the Data Privacy Strategy section: https://the-turing-way.netlify.app/project-design/sdpm/data-privacy-strategies.html
